api/tests/unit/domain/usecases/create-sessions_test.js

```diff
@@ -84,11 +92,115 @@ describe('Unit | UseCase | create-sessions', function () {
     });
 
     context('when the session has candidates', function () {
+      context('when certification center is not of type SCO', function () {
+        context('when certification candidate has complementary certification', function () {
+          it('should save certificationCandidate with complementary certification', async function () {
+            // given
+            const validSessionData = createValidSessionData();
+            const validCandidateData = createValidCandidateData();
+            const complementaryCertification = domainBuilder.buildComplementaryCertification({ label: 'Pix Toto' });
+            validCandidateData.complementaryCertifications = ['Pix Toto'];
+            const cpfBirthInformationValidation = CpfBirthInformationValidation.success({ ...validCandidateData });
+            certificationCpfService.getBirthInformation.resolves(cpfBirthInformationValidation);
+            const scoCertificationCenter = domainBuilder.buildCertificationCenter({
+              id: certificationCenterId,
+              type: 'SUP',
+            });
+            complementaryCertificationRepository.getByLabel.withArgs({ label: 'Pix Toto' }).resolves({
+              id: complementaryCertification.id,
+              key: complementaryCertification.key,
+              label: complementaryCertification.label,
+            });
```

Comment:
Est-ce qu'on peut pas plus simplement faire:
```suggestion
            complementaryCertificationRepository.getByLabel.withArgs({ label: 'Pix Toto' }).resolves(complementaryCertification);
```